ocp_resources/node_network_configuration_policy.py

```diff
@@ -329,7 +329,10 @@ def _absent_interface(self):
     @property
     def status(self):
         for condition in self.instance.status.conditions:
-            if condition["type"] == self.Conditions.Type.AVAILABLE:
+            if (
```

Comment:
This function returns the status as is.
this is not wait_for_status function

Reply:
it is not returning the whole condition, just the condition["reason"]. And is only used in ```ocp_resources/node_network_configuration_policy.py:        samples = TimeoutSampler(wait_timeout=self.success_timeout, sleep=1, func=lambda: self.status)
```